src/components/components/ebay-tooltip-base/constants.ts

```diff
@@ -0,0 +1,14 @@
+export const pointerStyles:{[index: string]: string} = {
```

Comment:
If you want to be more precise with this type, you can say that the value is `Placement`

Reply:
Good call... Ill swap it.